OpenDreamRuntime/Objects/MetaObjects/DreamMetaObjectMatrix.cs

```diff
@@ -14,6 +15,42 @@ sealed class DreamMetaObjectMatrix : IDreamMetaObject {
         public DreamMetaObjectMatrix() {
             IoCManager.InjectDependencies(this);
         }
+        public void OnObjectCreated(DreamObject dreamObject, DreamProcArguments creationArguments) {
+            if (creationArguments.Count > 0) {
```

Comment:
Testing showed that on a `matrix()` call, the argument count was 0. Named arguments aren't a concern I don't think, since they aren't valid?